docs/analysing_tardis_outputs/rpacket_plot_notebook.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Broken image link in ReviewNB</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/3003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>